SonarRunner.Shim/RoslynV1SarifFixer.cs

```diff
@@ -87,42 +96,57 @@ private static string ApplyFixToSarif(string unfixedSarif)
             return string.Join(Environment.NewLine, inputLines);
         }
 
-        /// <summary>
-        /// Used to correct invalid SARIF generated by VS 2015 RTM Roslyn. 
-        /// Correctly escapes backslashes and quotes.
-        /// 
-        /// Returns true if the output is valid SARIF (even if there was no change).
-        /// </summary>
-        public static bool FixRoslynV1Sarif(string unfixedSarif, out string output)
+        #endregion
+
+        #region IRoslynV1SarifFixer
+
+        public string LoadAndFixFile(string sarifPath, ILogger logger)
         {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) In member 'RoslynV1SarifFixer.LoadAndFixFile(string, ILogger)', change parameter name 'sarifPath' to 'sarifFilePath' in order to match the identifier as it has been declared in 'IRoslynV1SarifFixer.LoadAndFixFile(string, ILogger)'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=fxcop%3AParameterNamesShouldMatchBaseDeclaration)